spec/mocha/lib_spec.js

```diff
@@ -42,7 +42,7 @@ describe('protractor library', function() {
       browser.get('index.html').then(function() { finished = true; });
     });
 
-    after('verify mocha waited', function() {
+    after(function() {
```

Comment:
is this a change from Mocha? If not, we should continue to support a description for 'after' if it's a valid Mocha construct.